examples/dht_server.rs

```diff
@@ -200,6 +221,8 @@ fn main() {
     let server = add_server_main_loop(server, &server_obj);
     let server = add_onion_key_refresher(server, &server_obj);
     let server = server.join(dht_pk_handler).map(|_| ());
+    let server = server.join(lossless_handler).map(|_| ());
```

Comment:
Why did you chose join instead of select?

Reply:
https://docs.rs/futures/0.2.1/futures/trait.FutureExt.html#method.join - Joins the result of two futures, waiting for them both to complete.
https://docs.rs/futures/0.2.1/futures/trait.FutureExt.html#method.select - Waits for either one of two differently-typed futures to complete.

I need both futures but not any.

Reply:
And I think that `select` in other places should be changed to `join` so we will wait all the futures. Actually they are equal here since all these futures will never be complete but logically `join` fits better here.